sdk/highlight-go/tracer.go

```diff
@@ -93,7 +92,6 @@ func (t Tracer) InterceptResponse(ctx context.Context, next graphql.ResponseHand
 	start := graphql.Now()
 	resp := next(ctx)
 	end := graphql.Now()
-	t.log(ctx, span, resp.Errors)
```

Comment:
see description: `InterceptResponse` would be the aggregate error set for all fields in the request, meaning that it would duplicate errors (and not have full stacktraces for them).